lua/weapons/m9k_m61_frag.lua

```diff
@@ -31,7 +31,7 @@ SWEP.AdminSpawnable         = true
 SWEP.FiresUnderwater        = true
 
 SWEP.Primary.Sound          = ""
-SWEP.Primary.RPM            = 30 -- This is in Rounds Per Minute
+SWEP.Primary.RPM            = 54 -- This is in Rounds Per Minute
```

Comment:
this number was calculated by dividing 60 by the delay created by the timers (`1.1`) and seems to be close enough to the previous throwing delay.